spec/services/search_indexer_spec.rb

```diff
@@ -222,7 +228,7 @@ def scrub(html, strip_diacritics: false)
 
       post.rebake!
       post.reload
-      topic = post.topic
+      post.topic
```

Comment:
Same here too https://github.com/discourse/discourse/pull/10862/files#r501383631

Reply:
pushed changes for both